Web-Editor.md

```diff
@@ -12,16 +12,19 @@ Editor sessions are first created by executing a command on the server.
 You can run different commands to open sessions with different scopes:
 
 
-| Scope                            | Command                    |
-|----------------------------------|----------------------------|
-| **All groups and online users:** | `/lp editor`               |
-| **All groups:**                  | `/lp editor groups`        |
-| **All online users:**            | `/lp editor users`         |
-| **For a specific group:**        | `/lp group <group> editor` |
-| **For a specific user:**         | `/lp user <user> editor`   |
+| Scope                                          | Command                    |
+|------------------------------------------------|----------------------------|
+| **All groups and online users:**               | `/lp editor`               |
+| **All groups:**                                | `/lp editor groups`        |
+| **All online users:**                          | `/lp editor users`         |
+| **For a specific group:**                      | `/lp group <group> editor` |
+| **For a specific user:**                       | `/lp user <user> editor`   |
+| **For users only with a specific permission:** | `/lp editor users <filter>`|
 
 > Remember to use `/lpb` instead of `/lp` if you want to target your command to the BungeeCord version of LP and `/lpv` for Velocity.
 
+> The user permission filter command will load only users with a permission matching the filter to the editor.
```

Comment:
Maybe add the fact that it's only permissions the user has directly, and not permission inherited from groups.